dev-libs/qxlsx/qxlsx-1.3.56.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake qmake-utils
+
+DESCRIPTION="Excel file(*.xlsx) reader/writer library using Qt5"
+
+HOMEPAGE="https://github.com/QtExcel/QXlsx"
+SRC_URI="https://github.com/QtExcel/QXlsx/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="MIT"
+SLOT="0/${PV}"
```

Comment:
Does this also change ABI every minor release? Maybe 1.3 -> 1.4 makes more sense. But are you sure about that either?

Reply:
Without my patch it can't even be installed. I'll ask upstream in https://github.com/QtExcel/QXlsx/pull/114 though.